vue-frontend/src/components/jobs/LifeAtCanopas.vue

```diff
@@ -59,18 +60,65 @@ import AspectRatio from "@/components/utils/AspectRatio.vue";
 import SwiperCore, { Pagination, Autoplay } from "swiper";
 import { Swiper, SwiperSlide } from "swiper/vue";
 
-import life1 from "@/assets/images/life/jobs_canopas_life_1.webp";
-import life2 from "@/assets/images/life/jobs_canopas_life_2.webp";
-import life3 from "@/assets/images/life/jobs_canopas_life_3.webp";
-import life4 from "@/assets/images/life/jobs_canopas_life_4.webp";
-import life5 from "@/assets/images/life/jobs_canopas_life_5.webp";
-import life6 from "@/assets/images/life/jobs_canopas_life_6.webp";
-import life7 from "@/assets/images/life/jobs_canopas_life_7.webp";
-import life8 from "@/assets/images/life/jobs_canopas_life_8.webp";
-import life9 from "@/assets/images/life/jobs_canopas_life_9.webp";
-import life10 from "@/assets/images/life/jobs_canopas_life_10.webp";
-import life11 from "@/assets/images/life/jobs_canopas_life_11.webp";
-import life12 from "@/assets/images/life/jobs_canopas_life_12.webp";
+import life1_400w from "@/assets/images/life/jobs_canopas_life_1-400w.webp";
+import life1_800w from "@/assets/images/life/jobs_canopas_life_1-800w.webp";
+import life1_1600w from "@/assets/images/life/jobs_canopas_life_1-1600w.webp";
```

Comment:
Why not used 1200w images?

Reply:
Means instead of 1600w image or create new image with 1200 width ??

Reply:
I am just asking the reason, why you haven't added a 1200w image.

Reply:
Accha, because life at canopas image aspect ratio is 16:9 so, that's why using 1600 width image and in portfolio responsive image standard is 400w,800w,1600w and 2400w